.eslintrc

```diff
@@ -1,19 +1,67 @@
 {
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings"
+  ],
+  "plugins": [
+    "react"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
   "env": {
+    "es6": true,
     "browser": true,
     "node": true,
-    "es6": true
+    "jquery": true,
```

Comment:
why are you using jquery?

Reply:
Just for the Bootstrap JavaScript plugin

Reply:
why not use react-bootstrap?

Reply:
this will potentially mess up your react shadow dom diffing algorithm